src/configure/configure.ts

```diff
@@ -100,22 +103,25 @@ class Orchestrator {
     public async configure(node: any): Promise<void> {
         telemetryHelper.setCurrentStep('GetAllRequiredInputs');
         await this.getInputs(node);
-
         if (this.continueOrchestration) {
-            let pipelineConfigurer = ConfigurerFactory.GetConfigurer(this.inputs.sourceRepository, this.inputs.azureSession,
-                this.inputs.pipelineConfiguration.template.templateType, this.localGitRepoHelper);
-            let selectedCICDProvider = (pipelineConfigurer.constructor.name === "AzurePipelineConfigurer") ? constants.azurePipeline : constants.githubWorkflow;
+            if (this.inputs.sourceRepository.repositoryProvider === RepositoryProvider.Github &&
+                this.inputs.targetResource.resource.type == TargetResourceType.AKS ) {
+               return await this.configurePipelineRemotely();
```

Comment:
This new feature should be behind a feature flag so that we can turn it off immediately if users report major issues with it

Reply:
For reference see `templateServiceEnabled` in `ExtensionVariables` class

Reply:
Added feature flag deployToAzure.UseRemoteConfigurer